scripts/export/v0_33_0/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	app "github.com/cosmos/cosmos-sdk/cmd/gaia/app"
+	"github.com/cosmos/cosmos-sdk/scripts/export"
+)
+
+// Command: go run main.go [path_to_old_genesis.json] [chain-id] [genesis-start-time] > [path_to_new_genesis.json]
+func main() {
+	cdc := app.MakeCodec()
+
+	args := os.Args[1:]
+	if len(args) != 3 {
+		panic(fmt.Errorf("please provide path, chain-id and genesis time"))
+	}
+
+	pathToGenesis := args[0]
+	chainID := args[1]
+	genesisTime := args[2]
+
+	err := export.ValidateInputs(pathToGenesis, chainID, genesisTime)
+	if err != nil {
+		panic(err)
+	}
+
+	genesis, err := export.NewGenesisFile(cdc, pathToGenesis)
+	if err != nil {
+		panic(err)
+	}
+
+	genesis.ChainID = strings.Trim(chainID, " ")
```

Comment:
imho, we should hardcode the chain-id here. What's the rationale for it being user input?

Reply:
oh, just because I don't know which is the exact chainID we'll use. Will it be `cosmoshub-2` ?

Reply:
updated